Compute/src/V1/Gapic/AcceleratorTypesGapicClient.php

```diff
@@ -412,7 +412,7 @@ public function get($acceleratorType, $project, $zone, array $optionalArgs = [])
      *
      * @throws ApiException if the remote call fails
      */
-    public function list_($project, $zone, array $optionalArgs = [])
+    public function list($project, $zone, array $optionalArgs = [])
```

Comment:
We must revert this change before merge as it should happen upstream in the GAPIC generator (see https://github.com/googleapis/gapic-generator-php/pull/292)

Reply:
Feel free to merge this once  that is done 👍